stats/release_queue.php

```diff
@@ -109,8 +109,9 @@ function _show_round_queues($round, $listing_view_mode)
         echo "<th>", html_safe(_("Current length")), "</th>\n";
         echo "<th>", html_safe(_("Length without holds")), "</th>\n";
         if (user_can_see_queue_settings()) {
-            echo "<th>", html_safe(_("Release criterion")), "</th>\n";
-            $columns += 1;
+            echo "<th>", html_safe(_("Projects target")), "</th>\n";
```

Comment:
We might consider formatting these as expressions for display (here and on the queue detail page) rather than just displaying the numbers in columns. I find `projects < 30 or pages < 200` more intuitive. The formatting function could do special things for some of our edgecases like projects < $MAGIC_NUMBER.

Reply:
Done. It occurs to me that we could just add
```
"release_criterion" => format_queue_targets_as_condition(...)
```
in `_queue_data()`, and then `release_queue.php` wouldn't have to change from the status quo at all. It depends whether you want that 'computed field' to appear everywhere that calls `_queue_data`.